data_steward/test/unit_test/data_steward/cdr_cleaner/manual_cleaning_rules/clean_smoking_ppi_test.py

```diff
@@ -0,0 +1,44 @@
+import csv
```

Comment:
This will eventually become an integration test because it is actually loading data into big query and querying it.  It's fine where it is for now.